app/internal/whatsapp.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional
+from urllib.parse import urlencode
+
+
+def make_link(phone_number: Optional[str], message: Optional[str]) -> str:
```

Comment:
This should have a route, like `/event/share/whatsapp` :)

I want to see this function in the [API](https://calendar.pythonic.guru/docs).
It will be automatically added when you add a route using FastAPI.